docs/next/solito.mdx

```diff
@@ -1,4 +1,6 @@
-For a long long time the React Native community has been looking for a way to make Universal Apps a reality, and the major roadblock that was stopping us was Navigation. Solito is most probably the answer to that, and as we know NativeBase already works on all the three platforms. It’s a match made in heaven. So now your next can run with the same codebase on all the 3 platforms and we are here to show you how?
+For a long long time, the React Native community has been looking for a way to make Universal Apps a reality. The major roadblock that was stopping us was Navigation. Solito is most probably the answer. And as we know, NativeBase already works on all three platforms. This makes the NativeBase and Solito combination a match made in heaven.
+
+With it, your next can run with the same codebase on all three platforms. Here is how to proceed.
```

Comment:
With it, your next application can run with the same codebase on all three platforms. Here is how to proceed.